src/components/EmptyState/EmptyState.tsx

```diff
@@ -16,12 +16,11 @@ export interface EmptyStateProps {
   /** The empty state heading */
   heading?: string;
   /** The path to the image to display */
+  /** The image should have ~40px of white space above when empty state is used within a card, modal, or navigation component */
```

Comment:
These comments are used for documentation in the styleguide. I am not sure what will happen with two comments.

Reply:
Good call. It doesn't look like we do this anywhere. I'll wrap as one quote